contracts/staking/StakingPoolFactory.sol

```diff
@@ -34,12 +34,7 @@ contract StakingPoolFactory is RoleAccessControl, PausableUpgradeable, Version {
         StakingPoolLib.Config calldata config,
         bool launchPaused,
         uint32 rewardsAvailableTimestamp
-    )
-        external
-        whenNotPaused
-        atLeastDaoAdminRole(config.daoId)
-        returns (address)
-    {
+    ) external whenNotPaused returns (address) {
```

Comment:
@CjHare Is it ok to have this function unpermissioned in terms of access control. The equivalent `createBond()` in the BondFactory is unpermissioned, just addition of the Bond to the Dao's set of created bonds is permissioned.